service-runner.js

```diff
@@ -100,6 +102,7 @@ ServiceRunner.prototype._sanitizeConfig = function(conf, options) {
         // use the number of CPUs
         conf.num_workers = os.cpus().length;
     }
+    conf.timeout = conf.timeout || 1000;
```

Comment:
IMHO, *timeout* is too generic a parameter name. Perhaps *heartbeat_timeout* or something similar would be more appropriate.